locales/en/global.json

```diff
@@ -0,0 +1,6 @@
+{
+    "global_banner_new_service": "This is a new service - your ",
+    "global_banner_feedback": "feedback",
```

Comment:
Would you be ok to change this one to be "global_banner_feedback_link"

Reply:
yep, done